src/install/apache/whatsNext.mdx

```diff
@@ -0,0 +1,637 @@
+        Enable multi-tenancy monitoring.
+      </td>
+
+      <td>
+        true
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        Metrics, inventory
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `METRICS`
+      </td>
+
+      <td>
+        Set to `true` to enable metrics-only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `INVENTORY`
+      </td>
+
+      <td>
+        Set to `true` to enable inventory-only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+  </tbody>
+</table>
+
+## Example configurations [#examples]
+
+Here are some example YAML configurations: 
+
+<CollapserGroup>
+  <Collapser
+    id="example1"
+    title="Basic configuration"
+  >
+    This is a very basic configuration to collect metrics and inventory from your localhost:
+
+    ```yml
+    integrations:
+      - name: nri-apache
+        env:
+          METRICS: "true"
+          STATUS_URL: http://127.0.0.1/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+
+      - name: nri-apache
+        env:
+          INVENTORY: "true"
+          STATUS_URL: http://127.0.0.1/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 60s
+        labels:
+          env: production
+          role: load_balancer
+        inventory_source: config/apache
+    ```
+  </Collapser>
+
+  <Collapser
+    id="example2"
+    title="HTTP basic authentication"
+  >
+    This configuration collects metrics and inventory from your localhost protected with basic authentication. Replace the `username` and `password` on the `STATUS_URL` with your credentials:
+
+    ```yml
+    integrations:
+      - name: nri-apache
+        env:
+          METRICS: "true"
+          STATUS_URL: http://username:password@127.0.0.1/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+
+      - name: nri-apache
+        env:
+          INVENTORY: "true"
+          STATUS_URL: http://username:password@127.0.0.1/server-status?auto
+          REMOTE_MONITORING: true
+        interval: 60s
+        labels:
+          env: production
+          role: load_balancer
+        inventory_source: config/apache
+    ```
+  </Collapser>
+
+  <Collapser
+    id="example3"
+    title="Custom Apache binary location"
+  >
+    By default the integration tries to detect the Apache binary on these 2 different locations: `/usr/sbin/httpd` and `/usr/sbin/apache2ctl`.<br/>If your Apache is installed on a custom location or you are using MS Windows, use the `BINARY_PATH` setting to specify the correct location and filename of your Apache binary:
```

Comment:
i am not sure how i feel about the unofficial break `<br/>` no action here i am just expressing my skepticism. why not use a traditional ~ break